### ISSUE TITLE
Preparations for the release of snippetsForNVDA 1.0.9

### ISSUE THREAD

**Issue by thgcode** [thgcode/snippetsForNVDA#25]
Hello!

SnippetsForNVDA 1.0.9 will be the first version that will require a small translation update, to add the text for the load and save snippets command.

The code is currently unreleased, to not break the translation for users, but it is merged on master in preparation for the new release. I'll send the message to the translators that translated the previous version, to not break the other translations.

For Brazilian Portuguese, I can update it as it is my first language, but @VovaMobile @cary-rowen when you have the chance, could you have a look?

Thank you for the help!

**Comment by cary-rowen** [comment]
Yes, I will update the translation, and I'd also like to contribute some other features to it.
